alohomora/src/rocket/form.rs

```diff
@@ -273,6 +273,76 @@ macro_rules! impl_form_via_rocket {
     )+)
 }
 
+macro_rules! impl_form_via_rocket_prim {
```

Comment:
@alex-douk what does this code do? did you write it or is it taken from some other commits we made to Sesame since you branched out?

Reply:
I mean, I understand what it does, it makes it so that these base types (e.g. i32) is parsable from GET and POST request parameters
I am just confused how our applications worked previously without this code since they all pass various Strings and ints via HTTP requests

Reply:
OH Now I understand.

The code on main (which also exists on your branch) is to implement this trait for BBox<T, P> where T is primitive types (i32 and the likes)
The code you added implements the trait directly for T.
This is intentionally not a thing we did. If an application developer wants to receive some unprotected data via an HTTP request, they have to explicity type it as BBox<T, NoPolicy>.

I will not add this code to main in the merge. You will likely need to change your application code so that it has an explicit NoPolicy in places where data is unpolicied.